scripts/updateReadme.js

```diff
@@ -0,0 +1,23 @@
+const fs = require("fs");
+
+const excalidrawDir = `${__dirname}/../src/packages/excalidraw`;
+let data = fs.readFileSync(`${excalidrawDir}/README_NEXT.md`, "utf8");
+
+// remove note for unstable release
+data = data.replace(
+  /<!-- stable-readme-start-->[\s\S]*?<!-- stable-readme-end-->/,
```

Comment:
shouldn't we have named this `unstable-readme-start`? 😀

Reply:
yes indeed :D